syncd/RequestShutdownCommandLineOptionsParser.cpp

```diff
@@ -37,7 +39,7 @@ std::shared_ptr<RequestShutdownCommandLineOptions> RequestShutdownCommandLineOpt
     {
         int option_index = 0;
 
-        int c = getopt_long(argc, argv, "cwfpg:x:h", long_options, &option_index);
+        int c = getopt_long(argc, argv, "cwfepzg:x:h", long_options, &option_index);
```

Comment:
since we have so many different boot types it make more sens to have single flag like -b "cold|warm|fast..." instead of giving new flag for each paramete,r but it will be hard to move from this since those flags are used all over the place ..

Reply:
agree. It will be an effort to do cleanup...